pandas/tests/io/formats/style/test_to_latex.py

```diff
@@ -484,8 +484,160 @@ def test_parse_latex_css_conversion(css, expected):
     assert result == expected
 
 
+@pytest.mark.parametrize("environment", ["tabular", "longtable"])
+def test_parse_latex_css_convert_minimal(styler, environment):
+    # parameters ensure longtable template is also tested
+    styler.highlight_max(props="font-weight:bold;")
+    result = styler.to_latex(convert_css=True, environment=environment)
+    assert (
```

Comment:
These multiline asserts looks quite cumbersome.
Could you consider extracting a variable, like ``chunk`` or something and asserting ``assert chunk in result``?

Reply:
done (and paramterized)